packages/flutter_test/lib/src/_matchers_io.dart

```diff
@@ -36,17 +36,20 @@ Future<ui.Image> captureImage(Element element) {
 /// test is running on a VM using conditional import.
 class MatchesGoldenFile extends AsyncMatcher {
```

Comment:
Does matchers web need these updates as well?

Reply:
Yes thank you!

I am not as familiar with how the web executes comparison. I think this will work.